abis/linux/vm-flags.h

```diff
@@ -13,18 +15,23 @@
 #define MAP_FIXED     0x10
 #define MAP_ANON      0x20
 #define MAP_ANONYMOUS 0x20
+#define MAP_NORESERVE 0x4000
```

Comment:
This does not seem to be defined by POSIX, but rather to be Linux-specific?

Reply:
Fixed.